jc/lib.py

```diff
@@ -238,14 +239,17 @@
     'zpool-status'
 ]
 
+
```

Comment:
It looks like you added some additional newlines to the `lib.py` file. Could you back those out? We want the PR to be only about `amixer` changes.

Reply:
@kellyjonbrazil I've added those new lines to the `jc/lib.py` module because of the `PEP 8: E302` violation, as part of boy scout principle, I've done it "on the way", I could open another issue/submit a dedicated PR.

Reply:
Yeah that makes sense, but we want to keep only relevant changes in the PR, which will make this easier to reason about in five years. 😄 This shows the specific intent of this PR is to add this parser. We can make the formatting changes in a separate PR.

Reply:
reverted the specific commit (The hash of that commit `241d1a1c630bd21ef7bd443d55a3eb149d77c943`), I've created a dedicated commit for that for easy reverting :) i knew that you might not like that.